pkg/curatedpackages/packagecontrollerclient.go

```diff
@@ -74,6 +77,12 @@ func NewPackageControllerClient(chartInstaller ChartInstaller, kubectl KubectlRu
 //   - credentials refreshing cron job creation
 //   - activation of a curated packages bundle
 func (pc *PackageControllerClient) InstallController(ctx context.Context) error {
```

Comment:
We need to update the comment above
Also should we rename this method? Like `EnableCuratedPackages` (avoiding using `install` to remove confusion with installing the actual packages). It doesn't just install the controller, now it depends on what type of cluster

Reply:
Yes I like EnableCuratedPackages